public/locales/en/common.json

```diff
@@ -82,8 +82,12 @@
   "newcomer" : "New to MAPLE? For extra support, check",
   "newsfeed": {
     "action_update": "has an action update from the ",
+    "and": " and ",
+    "not_follow_either": "You do not follow either",
```

Comment:
micro nit: We should probably consistently use camelCase for the translation keys.

Reply:
done